lamindb/models/history.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+
+
+class History(models.Model):
```

Comment:
I have a `created_by` field in the history table here:

- https://github.com/laminlabs/lamindb/pull/2640

Doesn't have to be part of this version. Just FYI.

Reply:
I'm not sure the trigger can extract `created_by`, especially since both SQLAlchemy-driven and Django-driven operations can modify the tables' state. Let me know if there's an easy way to do that that I'm missing.

Reply:
We can likely get that from connection/transaction that's live that defines that variable. But let's not worry about it right now. Will be easy to add down the road.